webtool/frontend/src/components/main-content.tsx

```diff
@@ -18,13 +19,6 @@ export const MainContent: FunctionComponent<EmotionProps> = () => {
         setShowSimulation(false)
     }
 
-    useEffect(() => {
-        // Set Default HouseholdGroup, minimum required to get run the simulation
-        const defaultHouseholdGroup = new HouseholdGroup("Huishoudens", 200, 0.2, 0.1, 0.2, 0.0, 3000);
```

Comment:
I think after this fix, the simulation was fixed to run without setting up the minimum required. Did you test that the simulation won't be an error if no household is defined?

Reply:
The web app still initializes a household group, but the simultation works without.